smartcar/vehicle.py

```diff
@@ -442,6 +442,43 @@ def unsubscribe(self, amt: str, webhook_id: str) -> types.Status:
         response = helpers.requester("DELETE", url, headers=headers)
         return types.select_named_tuple("unsubscribe", response)
 
+    # ===========================================
+    # General Purpose Request Method
+    # ===========================================
+
+    def request(
+        self, method: str, path: str, body: dict = {}, headers: dict = {}
+    ) -> types.Response:
+        """
+        Utility method to make a request to a Smartcar endpoint - can be used
+        to make requests to brand specific endpoints.
+
+        Args:
+            method (str): The HTTP request method to use.
+            path (str): The path to make the request to.
+            body (dict): The request body.
+            headers (dict): The headers to inlcude in the request.
+
+        Returns:
+            Response = NamedTuple("Response", [("body", dict), ("meta", dict)])
```

Comment:
if we change the definition above for the `meta` type, make sure we update it here too